Content.Server/GameTicking/GameTicker.cs

```diff
@@ -531,15 +534,15 @@ public bool PauseStart(bool pause = true)
 
             if (pause)
             {
-                _pauseTime = DateTime.UtcNow;
+                _pauseTime = IoCManager.Resolve<IGameTiming>();
             }
             else if (_pauseTime != default)
             {
-                _roundStartTimeUtc += DateTime.UtcNow - _pauseTime;
+                _roundStartTimeTicks += IoCManager.Resolve<IGameTiming>().RealTime.Ticks - _pauseTime.RealTime.Ticks;
             }
 
             var lobbyCountdownMessage = _netManager.CreateNetMessage<MsgTickerLobbyCountdown>();
-            lobbyCountdownMessage.StartTime = _roundStartTimeUtc;
+            lobbyCountdownMessage.StartTime = new DateTime(_roundStartTimeTicks + _roundInitializedTimeUtc.Ticks, DateTimeKind.Utc);
```

Comment:
I've already said this before, don't use `DateTime`. Change that message to use `TimeSpan`.